From 3f474ce5939d6d00950abda01c6d0d774f91ff09 Mon Sep 17 00:00:00 2001
From: David Mercado <david.mercado@windriver.com>
Date: Wed, 20 Nov 2013 17:00:54 -0500
Subject: [PATCH 150/175] LSI ACP34XX: USB: Add acp-usb to list of compatible devices

git.yoctoproject.org/git/linux-yocto-3.10
commit 3b5b54b7ec578a9671f32c68537da2fb020c652b standard/axxia/base.

Some bootloaders use "acp-usb" for USB0's device name.  Add it to
the ehci-ci13612 driver's table of compatible devices.

Signed-off-by: Hong H. Pham <hong.pham@windriver.com>
---
 drivers/usb/host/ehci-ci13612.c |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/drivers/usb/host/ehci-ci13612.c b/drivers/usb/host/ehci-ci13612.c
index ff77ee7..8e4a1f3 100644
--- a/drivers/usb/host/ehci-ci13612.c
+++ b/drivers/usb/host/ehci-ci13612.c
@@ -318,7 +318,11 @@ static struct of_device_id ci13612_match[] = {
 		.type	= "usb",
 		.compatible = "lsi,acp-usb",
 	},
-	{},
+	{
+		.type	= "usb",
+		.compatible = "acp-usb",
+	},
+	{ /* end of list */ },
 };
 
 static struct platform_driver ci13612_ehci_driver = {
-- 
1.7.1

